### PR TITLE
Switch to the logging library and away from a custom logger

### LINKED ISSUE #3: Logging
Currently all logging is done with print statements. Using the python logger would remove the need for the custom timestamp functionality and allow more customised behaviour.

### PR DESCRIPTION
**Pull request by PalmTreeSoftware**
Using the logger library
Removal of print statements
Removal of custom logger code

### CODE REVIEW

File: updater.py [hunk 1]
```diff
@@ -17,19 +18,21 @@
 parser.add_argument("-v", "--verbose", help="Verbose", action="store_true")
 args = parser.parse_args()
 
+if args.verbose:
+    logging.basicConfig(format='%(asctime) s%(levelname)s:%(message)s', level=logging.DEBUG, datefmt='%m/%d/%Y %I:%M:%S')
```
Comment (PalmTreeSoftware) [inline updater.py]: Testing comment, this if statement could be simplified with the logger defaults being set then just adjusting the log level in the if statement. Reduces the risk of deviation in log format between different log levels

**Review by PalmTreeSoftware — COMMENTED** [review]
Self review, possible cleanup identified on the logger setup

File: updater.py [hunk 2]
```diff
@@ -17,19 +18,21 @@
...
+else:
+    logging.basicConfig(format='%(asctime) s%(levelname)s:%(message)s', level=logging.INFO, datefmt='%m/%d/%Y %I:%M:%S')
```
Comment (UnlimitedLtd) [inline updater.py]: Can we get a separator between the timestamp and the logging level?

File: godaddyddns/godaddy.py [hunk 3]
```diff
@@ -53,7 +53,11 @@ def get_a_record(self, domain: str) -> ARecord:
             timeout=self.timeout,
             headers=self.headers,
         )
-        self.printer(f"{response.request.url} {response.status_code}")
+        logger.debug(
+            "Request url: {}, Status Code: {}".format(
```
Comment (UnlimitedLtd) [inline godaddyddns/godaddy.py]: Please use f strings as per Google style docs

File: godaddyddns/godaddy.py [hunk 4]
```diff
@@ -85,6 +89,10 @@ def update_a_record(self, domain: str, ip: str, ttl: int = 600) -> None:
             headers=self.headers,
             json=[{"data": ip, "ttl": ttl}],
         )
-        self.printer(f"{response.request.url} {response.status_code}")
+        logger.debug(
+            "Request url: {}, Status Code: {}".format(
```
Comment (UnlimitedLtd) [inline godaddyddns/godaddy.py]: Please use f strings as per Google style docs

Comment (UnlimitedLtd) [reply godaddyddns/godaddy.py]: e.g. logger.debug(f"Request url: {response.request.url}, Status Code: {response.status_code}")

I guess whilst at it, url should be capitalised as well.

File: updater.py [hunk 5]
```diff
@@ -19,15 +19,26 @@
 args = parser.parse_args()
 
 
-verbose = utils.Verbose(verbose=args.verbose)
+logging.basicConfig(
```
Comment (UnlimitedLtd) [inline updater.py:22]: Please use f strings.

**Review by UnlimitedLtd — APPROVED** [review]

**Pull request merged**